recipes/cookiecutter-django-core/meta.yaml

```
@@ -0,0 +1,98 @@
+{% set name = "cookiecutter-django-core" %}
+{% set version = "2023.09.23" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cookiecutter/cookiecutter-django/archive/{{ version }}.tar.gz
+  sha256: 1a0c56f948a6a7f6b01c24e7cc13664f76e80724642bb1db12a5931d9bea7140
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - pip
+  run:
+    - python >=3.8,<4.0
+# Django
+# ------------------------------------------------------------------------------
+    - django >=4.2.0,<5.0
+# https://github.com/cookiecutter/cookiecutter-django/blob/master/requirements.txt
+# ------------------------------------------------------------------------------
+    - cookiecutter >=2.3.1
+    - sh >=2.0.6  # [not win]
```

What you have there is correct too. I just want to test why our linter is still failing here. The fix already already merged and I believe it should be fixed by now.

```suggestion
    - sh >=2.0.6  # [unix]
```